ProcessMaker/Http/Controllers/TaskController.php

```diff
@@ -11,12 +11,12 @@ public function index()
   {
       return view('tasks.index');
   }
-//     public function edit(Tasks $tasks) /*<-----*/ // 
-//   {
-//     return view('tasks.edit',compact($tasks));  //is the edit page added to routes?
-//   }
-//   public function create()
-//   {
-//     return view('tasks.create');
-//   }
+  public function create()
```

Comment:
Remove create. This method is index and show only

Reply:
removed